### DISCUSSION TITLE
Can Anyone Help About How Can I Setup Godot-Go On Mac

### DISCUSSION THREAD

**Discussion by temidaradev** [godot-go/godot-go#107] [Q&A] [answered]
I try to use Godot-go but I didn't understand how can I set up this extension

**Comment by pcting** [comment, marked as answer]
i'll try and improve the documentation at some point. you need to do the following:
1. install godot. remember the path of where the binary is.
2. install the build tooling such as `make` and `clang-format`.
3. try and see if you can generate the headers from the godot binary: `GODOT=$(which godot) make update_godot_headers_from_binary`

this should get your started
